README.md

```diff
@@ -76,10 +120,27 @@ npm install @codesignal/meteor-protomongo
 
 After the package is installed, add the following few lines in a file that's going to be loaded on startup:
```

Comment:
Do you think it would make sense to just ask the consumer to directly pass in the result of a `find()` instead, so that we can inject & remove that dependency?

The downside is that it exposes this "hack" to the consumer a little bit more. The upside is that it removes this dependency on `accounts-base`, which you're right is a little weird (I was trying to pick a collection I expect to exist in every Meteor instance, but I guess you're right that it's still a dep)

Reply:
Update: Okay, I went ahead and made this change to ask the consumer to pass in a cursor, rather than assuming that `Meteor.users` is a valid collection. Thanks for pointing out the implicit dependency!